lib/page_renderer.js

```diff
@@ -11,7 +11,6 @@ module.exports = class PageRenderer {
 	}
 
 	async render(page) {
-		await page.load();
```

Comment:
You don't wanna remove this; it's (almost) a no-op if the page instance was previously loaded/populated, but enables the method to be used regardless of the instance's previous state.

Reply:
Done.